src/header/value.rs

```diff
@@ -289,10 +289,33 @@ impl AsRef<[u8]> for HeaderValue {
 
 impl fmt::Debug for HeaderValue {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("HeaderValue")
-            .field("value", &EscapeBytes(self.as_ref()))
-            .field("is_sensitive", &self.is_sensitive)
-            .finish()
+        if self.is_sensitive {
+            f.write_str("Sensitive")
```

Comment:
That's what I was thinking yeah, basically just having both the sensitive and non-sensitive versions have roughly the same shape